stories/form/form-select/form-select.stories.mdx

```diff
@@ -72,6 +73,54 @@ import { Form } from "@lifesg/react-design-system/form";
     </Story>
 </Canvas>
 
+<Heading3>With custom display</Heading3>
+
+You are able to customise the rendering of the selected option and the dropdown items.
```

Comment:
**Edit**:
You are able to customise the display of the dropdown list items as well as their selected state.